dff/core/engine/conditions.py

```diff
@@ -211,7 +214,11 @@ def true_handler(ctx: Context, actor: Actor, *args, **kwargs) -> bool:
 
 @validate_arguments
 def false(*args, **kwargs) -> Callable:
```

Comment:
Should we add Callable signature (☝️): `Callable[[Context, Actor, Any, Any], bool]` (AFAIK) here.